src/main/target/SPEEDYBEEF4/target.h

```diff
@@ -138,24 +155,22 @@
 #define VBAT_ADC_CHANNEL                ADC_CHN_2
 #define RSSI_ADC_CHANNEL                ADC_CHN_3
 
-#define SENSORS_SET (SENSOR_ACC|SENSOR_MAG|SENSOR_BARO)
+/*** LED STRIP ***/
 
-#define DEFAULT_RX_TYPE         RX_TYPE_PPM
-#define DISABLE_RX_PWM_FEATURE
+/*** Default settings ***/
 #define DEFAULT_FEATURES        (FEATURE_TX_PROF_SEL | FEATURE_CURRENT_METER | FEATURE_BLACKBOX | FEATURE_VBAT | FEATURE_OSD)
+#define DEFAULT_RX_TYPE         RX_TYPE_SERIAL
+#define SERIALRX_PROVIDER       SERIALRX_SBUS
+#define CURRENT_METER_SCALE     166
```

Comment:
Previously we had default value of 302, why this is changed?

Reply:
The old value was way too high atleast for me. I've flown it really often on the stock 2.4.0 firmware and had around 2500mAh drawn, recharged about 4000mAh. On a brand new Turnigy NanoTech 4S LiPo. Checked the behaviour with other LiPos in my bag and the values i calculated for me are the same. That's why i changed it.

Reply:
The old value was way too high atleast for me. I've flown it really often on the stock 2.4.0 firmware and had around 2500mAh drawn, recharged about 4000mAh. On a brand new Turnigy NanoTech 4S LiPo. Checked the behaviour with other LiPos in my bag and the values i calculated for me are the same. That's why i changed it.